cross-modal-search/flows/flow-query.yml

```diff
@@ -17,8 +18,9 @@ pods:
 #    needs: loader
   - name: image_encoder                   # encode query image into embeddings with CLIP model
     polling: any
-    uses: 'jinahub+docker://ClipImageEncoder'
-    volumes: $HOME/.cache/clip:/root/.cache/clip
+     # uses: 'jinahub+docker://ClIPImageEncoder'
+     # volumes: $HOME/.cache/clip:/root/.cache/clip
```

Comment:
same

Reply:
Addressed, please see [e11b980](https://github.com/jina-ai/examples/pull/672/commits/e11b980488645a32eac1fbe3a7c9ca27def8d850) for the change.